src/main/java/org/sagebionetworks/bridge/exporter/handler/SynapseExportHandler.java

```diff
@@ -392,4 +398,22 @@ public void uploadToSynapseForTask(ExportTask task) throws BridgeExporterExcepti
     /** Creates a row values for a single row from the given export task. */
     protected abstract Map<String, String> getTsvRowValueMap(ExportSubtask subtask) throws BridgeExporterException,
             IOException, SchemaNotFoundException, SynapseException;
+
+
+    /**
+     * dummy method to implement by healthDataExportHandler to handle update record exporter status
+     * @param tsvInfo
+     * @throws BridgeExporterException
+     */
+    protected void postProcessTsv(TsvInfo tsvInfo) throws BridgeExporterException {
+
+    };
+
+    /**
+     * dummy method to implement by HealthDataExportHandler for testing
+     * @param bridgeHelper
+     */
+    protected void setBridgeHelper(BridgeHelper bridgeHelper) {
+
+    }
```

Comment:
I don't love these... do both ExportHandlers have to have all the same methods? Can these be removed somehow? These aren't declared in ExportHandler so it seems like it should be possible to remove them.